src/App.js

```diff
@@ -1,79 +1,162 @@
 import React, { Component } from "react";
-import logo from "./logo.svg";
 import "./App.css";
-import picture from "./mood-smiley-face.jpg";
+import picture1 from "./nidhi.jpg";
+import picture2 from "./Nidhi_Gretch.jpg";
+import picture3 from "./Nidhi_Farha.jpg";
+
+import thumbnail1 from "./nidhi.jpg";
+import thumbnail2 from "./Nidhi_Gretch.jpg";
+import thumbnail3 from "./Nidhi_Farha.jpg";
+
+import ImageGallery from "react-image-gallery";
+
 import {
   Row,
   Col,
   Image,
   Button,
   ButtonToolbar,
-  FormControl
+  FormControl,
+  FormGroup,
+  ControlLabel,
+  HelpBlock
 } from "react-bootstrap";
-import { ProfileImage } from "./component";
+
+import { CenterLines } from "./CenterLines";
+import { CreateTable } from "./ContactForm";
+
+import { FieldGroup } from "./ContactForm";
+
+import { FormButton } from "./ContactForm";
```

Comment:
you can condense all of these imports and do `import { CreateTable, FieldGroup, FormButton } ...`